smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -516,11 +630,7 @@ def doDeviceSync(){
 	log.trace "Doing Hue Device Sync!"
 	convertBulbListToMap()
 	poll()
-    try {
-		subscribe(location, null, locationHandler, [filterEvents:false])
-    } catch (all) {
```

Comment:
I'm assuming this try/catch is here because the platform used to have a problem with duplicate subscriptions at some time in the past. As of right now, we have handled this on the platform side, so the try/catch should be unnecessary.